docs/general/testing/web/index.md

```diff
@@ -0,0 +1,23 @@
+---
+uid: testing-web
+title: Testing Jellyfin Web Clients
+---
+
+## Weekly Unstable Builds
+
+Weekly unstable builds ship Jellyfin Server and Jellyfin Web in a complete package. Please refer to [Testing Jellyfin Server](/docs/general/testing/server/) for more info.
+
+## Testing PRs from Cloudflare Pages Deployments
+
+Each PR of [Jellyfin Web](https://github.com/jellyfin/jellyfin-web) and [Jellyfin Vue](https://github.com/jellyfin/jellyfin-vue) is automatically deployed to Cloudflare Pages. Please follow the steps below on how to test them.
+
+1. Find the PR you would like to checkout in the list of open PRs. [PRs for Jellyfin Web](https://github.com/jellyfin/jellyfin-web/pulls), [PRs for Jellyfin Vue](https://github.com/jellyfin/jellyfin-vue/pulls)
```

Comment:
```suggestion
1. Find the PR you would like to checkout in the list of open PRs ([PRs for Jellyfin Web](https://github.com/jellyfin/jellyfin-web/pulls), [PRs for Jellyfin Vue](https://github.com/jellyfin/jellyfin-vue/pulls))
```

Reply:
@felix920506 Any reason why this is not changed?

Reply:
Another approach, which makes more sense:
```suggestion
1. Find the PR you would like to checkout in the list of open PRs: [PRs for Jellyfin Web](https://github.com/jellyfin/jellyfin-web/pulls), [PRs for Jellyfin Vue](https://github.com/jellyfin/jellyfin-vue/pulls)
```

Reply:
I think the change you suggested actually makes it more cluttered which isn't something we want for a docs, but I don't have any ideas on how to change it either

Reply:
Does the two dots look good enough for you?

I understand the cluttering feeling, but imo it's important we stick with all the "writing conventions" and redaction it's important for an easier reading too!

Reply:
Looks good